packages/lesswrong/server/resolvers/subscribedUsersFeedResolver.tsx

```diff
@@ -0,0 +1,78 @@
+import { Posts } from "../../lib/collections/posts";
+import { Subscriptions } from "../../lib/collections/subscriptions";
+import { defineFeedResolver } from "../utils/feedUtil";
+import { addGraphQLSchema } from "../vulcan-lib";
+import { Comments } from "../../lib/collections/comments";
+import keyBy from "lodash/keyBy";
+import { PostsRepo } from "../repos";
+import { loadByIds } from "../../lib/loaders";
+import { filterNonnull } from "../../lib/utils/typeGuardUtils";
+import { accessFilterMultiple } from "../../lib/utils/schemaUtils";
+
+addGraphQLSchema(`
+  type SubscribedPostAndComments {
+    _id: String!
+    post: Post!
+    comments: [Comment!]!
+    postIsFromSubscribedUser: Boolean!
+  }
+`);
+
+defineFeedResolver<Date>({
+  name: "SubscribedFeed",
+  cutoffTypeGraphQL: "Date",
+  args: "af: Boolean",
+  resultTypesGraphQL: `
+    postCommented: SubscribedPostAndComments
+  `,
+  resolver: async ({limit=20, cutoff, offset, args, context}: {
+    limit?: number, cutoff?: Date, offset?: number,
+    args: {af: boolean},
+    context: ResolverContext
+  }) => {
+    const currentUser = context.currentUser;
+    if (!currentUser) throw new Error("Must be logged in");
+
+    const postsAndCommentsAll = await new PostsRepo().getPostsAndCommentsFromSubscriptions(currentUser._id, 30);
```

Comment:
you can access the posts repo from context without importing it: `context.repos.posts`